projects/netgrif-components/src/lib/data-fields/data-field-template/data-field-template.component.html

```diff
@@ -17,7 +17,7 @@
 
     <!-- Small layout -->
     <ng-template #elseTemplate>
-        <div fxLayout="row" fxLayoutAlign="start center" class="fill-all-space" [ngClass]="{'padding-8px': isNotButton()}">
+        <div fxLayout="row" fxLayoutAlign="start center" class="fill-all-space" [ngClass]="{'padding-8px': !isButton(), 'padding-16px': isButton()}">
```

Comment:
why we putting the 16px paddign there ? is the alignment of buttons with fields still ok ?

Reply:
@machacjozef @renczesstefan

Reply:
Can you please attach some screenshots for clearance?  🙏  Thank you

Reply:
@renczesstefan but you need to chech the alignment of buttons (if they are aligned with text field for example), there is a reason that the padding was removed

Reply:
This is how it looks like without this PR 
![image](https://github.com/netgrif/components/assets/45174598/436d4cad-ffcf-45cd-b09f-c5697b3719ce)

this is how it looks like with original padding but with no shadows
![image](https://github.com/netgrif/components/assets/45174598/ba2c23ae-3ede-4aef-afbf-ec023cc07803)

and this is how it looks like with new padding and with no shadow
![image](https://github.com/netgrif/components/assets/45174598/e4367d14-8850-44ee-8808-8dfcbec19b3f)

We should decide which one to chose.

Reply:
@tuplle & @Kovy95

Reply:
padding 8px from left and right and any padding from top and bottom